library/kani/src/tuple.rs

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Modifications Copyright Kani Contributors
```

Comment:
Is there a reason this is using the "modifed-file" license header as opposed to the standard one?

Reply:
No. I just copied the license header of whatever I had open next to it, which was likely the modified proptest. 

I'll change it back.

Reply:
72a48f243631eef863686f1726db395a867df4a0